.github/workflows/build-and-push.yml

```diff
@@ -209,8 +133,40 @@ jobs:
             docker cp assets-tmp:app/static/ ./$TMP_DIR/
             docker cp assets-tmp:app/static-upload-hashed ./$TMP_DIR_HASHED/
 
-            gsutil -m rsync -r ./$TMP_DIR_HASHED/* gs://$APP-${{ needs.push_image_to_gar.outputs.deployment_realm }}-${{ needs.push_image_to_gar.outputs.deployment_env }}-media/media/
-            gsutil -m rsync -r ./$TMP_DIR/* gs://$APP-${{ needs.push_image_to_gar.outputs.deployment_realm }}-${{ needs.push_image_to_gar.outputs.deployment_env }}-media/media/
+            gsutil -m rsync -r ./$TMP_DIR_HASHED/* gs://$APP-${{ needs.build_and_publish_public_images.outputs.deployment_realm }}-${{ needs.build_and_publish_public_images.outputs.deployment_env }}-media/media/
+            gsutil -m rsync -r ./$TMP_DIR/* gs://$APP-${{ needs.build_and_publish_public_images.outputs.deployment_realm }}-${{ needs.build_and_publish_public_images.outputs.deployment_env }}-media/media/
             rm -rf ./$TMP_DIR/ ./$TMP_DIR_HASHED/
             docker kill assets-tmp
             docker rm assets-tmp
+
+
+  push_image_to_gar:
+    name: Push bedrock image to GAR
+    needs: [build_and_publish_public_images, upload_static_assets]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - uses: docker/setup-buildx-action@v3
```

Comment:
TBH I still think this one shouldn't be here, but looking again IIUC it was always here even before, and that job was never supposed to build any of its own things even with the previous layout. So I'll keep it intact for now, the slowdown is not that much, and see about followup removal separately.

_(It may just spin up the builder container, do the dance with transferring contexts… and then sit there running idle. I'll keep an eye on it after this is running, to see if the attached builder is ever used for anything from this job.)_